### ISSUE TITLE
Background for cfw's today

### ISSUE THREAD

**Issue by joonro** [cpaulik/emacs-material-theme#29]
Hi,

When I have cfw's calender open, the today's color is `#E0F7FA` for the light theme and `1C1F26` for the dark one - and foreground loses all the colors. For example:

![clipboard image](https://cloud.githubusercontent.com/assets/1063143/13467149/7c551c88-e062-11e5-82fb-13e46c066129.jpg)

![clipboard image 1](https://cloud.githubusercontent.com/assets/1063143/13467182/a64d69b4-e062-11e5-9232-17135f8e40e3.jpg)

**Comment by cpaulik** [comment]
Hi,

I'm not sure what you mean exactly. The screenshots look OK to me.

**Comment by joonro** [comment]
I'm sorry - I have some personal things on the calendar so it is a bit challenging to generate an informative screenshot. For example, an item with a timestamp with time:

![clipboard image 1](https://cloud.githubusercontent.com/assets/1063143/13475245/641e3224-e087-11e5-9613-b173c68bf55b.jpg)

an item with a timestamp without time:

![clipboard image](https://cloud.githubusercontent.com/assets/1063143/13474992/17c666e0-e086-11e5-8ad5-e2c5ff06d130.jpg)

Basically all today's items do not have any color highlighting - they are all black with `#E0F7FA` background. For other dates, TODO items are highlighted with red, etc.

Losing highlighting for today's items make things harder to recognize - it would be great if `#E0F7FA` background color is gone, since today's date is already highlighted with `#BF616A`.

Please let me know if this is not clear, I will try to make a better screenshot.

**Comment by cpaulik** [comment]
I've removed the cfw:today background. But in my testing I still do not get proper highlighting when a timestamp is involved.

**Comment by joonro** [comment]
Thank you so much! I will test it and let you know.
